docs/team/botr99.md

```diff
@@ -1,46 +1,45 @@
-johndoe.md---
+---
```

Comment:
For this part im not sure whether need to put as botr99.md

Reply:
I don't think it will show up correctly on the website if I include it.

Reply:
Ok thanks!